core/io.rbs

```diff
@@ -1160,7 +1160,8 @@ class IO < Object
   #     f.gets(chomp: true) # => nil
   #     f.close
   #
-  def gets: (?String sep, ?Integer limit) -> String?
+  def gets: (String? sep, ?Integer limit, ?chomp: boolish) -> String?
```

Comment:
Slight nitpick: `gets` allows for implicit conversions, so we should be using `string?` and `int`.